### PR TITLE
remove un-needed marks

### PR DESCRIPTION
**Pull request by liubin**
Already in `Note` block.

### CODE REVIEW

File: content/en/docs/getting-started-guides/scratch.md [hunk 1]
```diff
@@ -493,12 +493,12 @@ You will need to run one or more instances of etcd.
     by durable storage (RAID, GCE PD).
     
     {{< note >}}
-    **Note:** May result in operations outages in case of instance outage.
+    May result in operations outages in case of instance outage.
     {{< /note >}}
```
Comment (ajaeger) [inline content/en/docs/getting-started-guides/scratch.md]: See https://kubernetes.io/docs/home/contribute/style-guide/ - this is correct. Leave the Note in.

File: content/en/docs/getting-started-guides/scratch.md [hunk 2]
```diff
@@ -493,12 +493,12 @@ You will need to run one or more instances of etcd.
     by durable storage (RAID, GCE PD).
     
     {{< note >}}
-    **Note:** May result in operations outages in case of instance outage.
+    May result in operations outages in case of instance outage.
```
Comment (ajaeger) [inline content/en/docs/getting-started-guides/scratch.md]: The display is indeed wrong - wrap it differently as shown in https://kubernetes.io/docs/home/contribute/style-guide/#common-shortcode-issues

**Comment by zacharysarah** [comment]
@liubin Bumping for updates.

**Comment by liubin** [comment]
Need not to modify, close.

**Pull request closed without merging**